packages/itwin/tree-widget/src/e2e-tests/TreeWidget.test.ts

```diff
@@ -5,27 +5,24 @@
 
 import type { Locator } from "@playwright/test";
 import { test } from "@playwright/test";
-import { initTreeWidgetTest, locateNode, selectTree, takeScreenshot, withDifferentDensities } from "./utils.js";
+import { initTreeWidgetTest, locateNode, selectTree, takeScreenshot } from "./utils.js";
 
-test.describe.skip("Widget", () => {
+test.describe("Widget", () => {
   let treeWidget: Locator;
 
   test.beforeEach(async ({ page, baseURL }) => {
     treeWidget = await initTreeWidgetTest({ page, baseURL });
-    await locateNode(treeWidget, "BayTown").getByRole("checkbox", { name: "Visible: All models are visible", exact: true }).waitFor();
+    await locateNode(treeWidget, "ProcessPhysicalModel").waitFor();
   });
 
-  withDifferentDensities(() => {
-    test("tree selector", async ({ page }) => {
-      await treeWidget.getByRole("combobox").click();
-      await page.getByRole("listbox").waitFor();
-      await takeScreenshot(page, treeWidget);
-    });
+  test("tree selector", async ({ page }) => {
+    await treeWidget.getByRole("combobox").click();
+    await takeScreenshot(page, treeWidget);
+  });
 
-    test("tree selector badge", async ({ page }) => {
-      await selectTree(treeWidget, "External sources");
-      await page.getByText("The data required for this tree layout is not available in this iModel.").waitFor();
-      await takeScreenshot(page, treeWidget);
-    });
+  test("tree selector badge", async ({ page }) => {
```

Comment:
The "before" test targeted the "badge" / "icon" of the `Select` item. This one doesn't show it - I guess we don't support icons on `Select` items at the moment? I would be great to make it clear, because it's very confusing to have a test that succeeds on a component that doesn't work as expected. IMO, we need a comment before the test is necessary to explain that. Also, I'm debating whether we should add `.skip` on it or not 🤔